backend/node/base.py

```
@@ -154,6 +161,201 @@ def get_balance(self, addr: Address) -> int:
+            if resp.status != 200:
+                self.logger.error(
+                    f"Failed to stop LLM module", body=body, status=resp.status
+                )
+            else:
+                self.logger.info(f"Stopped LLM module", body=body, status=resp.status)
+
+    async def start_module(
+        self,
+        module_type: typing.Literal["llm", "web"],
+        config: dict[str, typing.Any] | None,
+        extra: dict = {},
```

Ruff (0.14.6)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)